db/migrate/20181126184505_create_dishes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDishes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :dishes do |t|
+      t.string :name
+      t.text :description
+      t.integer :price
+      t.string :category
```

Comment:
Do you guys have a separate model for the categories how will you play this out when you want to create an association?

Reply:
Per our discussion, it's easier to remove this and we'll add it during the association PR instead. Thanks.

Reply:
It is removed